options.js

```diff
@@ -72,6 +74,46 @@ function fillVals() {
 			break;
 		}
 	}
+    
+    //RTK
+ 
+    function removeRtkStories(){
+        const dictGroups = document.getElementById('dict-kpr');
+        localStorage.removeItem("rtkFilename");
+        localStorage.removeItem("rtkStories");
+        dictGroups.classList.add("novisible");   
+        chrome.extension.getBackgroundPage().rcxMain.config.rtkStories = null;    
+		// remove styleSheetId				

```

Comment:
not required?

Reply:
(that is it's commented out so I wondered if it should just be deleted or if it should be revived?)